src/cryptenroll/cryptenroll.c

```diff
@@ -253,7 +260,29 @@ static int parse_argv(int argc, char *argv[]) {
                         r = parse_path_argument(optarg, /* suppress_root= */ true, &arg_unlock_keyfile);
                         if (r < 0)
                                 return r;
+
+                        arg_unlock_type = UNLOCK_KEYFILE;
+                        break;
+
+                case ARG_UNLOCK_FIDO2_DEVICE: {
+                        _cleanup_free_ char *device = NULL;
+
+                        if (streq(optarg, "auto")) {
+                                if (arg_enroll_type == ENROLL_FIDO2)
+                                        return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                                               "When enrolling a new FIDO2 device, `--unlock-fido2-device=auto` is unsupported. "
```

Comment:
Please don't use backticks. They **only** make sense in markdown, and are not good for quoting in other places. They are also special to the shell. Just use normal single quotes.